components/PartCard/index.js

```diff
@@ -4,22 +4,28 @@ import {
   PartCardCategory,
   PartCardImage,
   PartCardText,
+  ToggleInAssemblerButton,
 } from "./PartCard.styled.js";
 import StatusMarker from "../StatusMarker/index.js";
+import usePartStore from "../PartStore/UsePartStore.js";
 
 export default function PartCard({ part, isDetail }) {
+  function toggleInAssembler() {
+    usePartStore.getState().togglePartValue(part.uuid, "inAssembler");
+  }
+
   return (
     <PartCardFlexContainer direction={"column"} border={"blue"}>
-      <PartCardFlexContainer direction={"row"}>
+      <PartCardFlexContainer direction={"row"} justify={"flex-start"}>
```

Comment:
if you just use a string for a prop value you don't need the curly brackets  `{}`

Reply:
good advice! they are gone